tasks/javascript.js

```diff
@@ -134,7 +134,7 @@ gulp.task('scripts:bundle', () => {
             ))
 
             // if production build, rip out our console logs
-            .pipe(gulpif(config.isProduction,
+            .pipe(gulpif(config.isProduction && config.js.stripDebug,
```

Comment:
Might be a silly question but why do we need to have this new option? Shouldn't we strip console logs / debuggers from production code anyway? Or is there a reason we use them in production.

Reply:
It's a request from the OrderWeb team.  They want to have production type code with debug statements for when they release to QA.

Reply:
One thing that would be worth adding is the ability to set this config variable through an environment variable.  So something like `gulp --stripDebug=false` would set this to false (as I imagine that might be how they need to set it to be different on every release server).

Reply:
@ashleynolan Should I add that in this PR?